src/App.tsx

```diff
@@ -1,25 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+
+  const startIdx = (currentPage - 1) * perPage;
+  const endIdx = startIdx + perPage;
+  const start = startIdx;
+  const end = Math.min(endIdx, items.length);
```

Comment:
The `start` and `end` variables are redundant since they directly mirror `startIdx` and `endIdx` (with a boundary check for `end`). Consider using `startIdx` and `endIdx` directly in the JSX to simplify the code.